python/ray/serve/_private/replica.py

```diff
@@ -158,7 +158,7 @@ async def __init__(
             )
 
             # Indicates whether the replica has finished initializing.
-            self._init_finish_event = asyncio.Event()
+            self._replica_init_finish = False
```

Comment:
```suggestion
            self._initialized = False
```
no need to prefix it with "replica" given this whole file is `replica.py` :)